pom.xml

```diff
@@ -65,12 +65,19 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>org.jenkins-ci.plugins.aws-java-sdk</groupId>
-            <artifactId>aws-java-sdk-minimal</artifactId>
+            <groupId>io.jenkins.plugins.aws-java-sdk2</groupId>
+            <artifactId>aws-java-sdk2-core</artifactId>
+            <version>2.30.28-26.v649b_8df2f988</version>
         </dependency>
         <dependency>
-            <groupId>org.jenkins-ci.plugins</groupId>
-            <artifactId>apache-httpcomponents-client-4-api</artifactId>
+            <groupId>io.jenkins.plugins.aws-java-sdk2</groupId>
+            <artifactId>aws-java-sdk2-s3</artifactId>
+            <version>2.30.28-26.v649b_8df2f988</version>
+        </dependency>
+        <dependency>
+            <groupId>software.amazon.awssdk</groupId> <!-- TODO Jenkins API plugin -->
```

Comment:
Unclear based on the draft status of this PR whether the intention is to complete this TODO or to defer it.

Reply:
Pending https://github.com/jenkinsci/aws-java-sdk2-plugin/pull/67